### PR TITLE
Local configuration prompter

### PR DESCRIPTION
**Pull request by hugodecasta**
# Local Config Prompt System
### affected systems
 - common.js
 - all config files
#### new module
 - added module [env-config-prompt](https://www.npmjs.com/package/env-config-prompt)
#### file change
 - config files
 - common.js config loader
#### advice for testers
 - remove the localConfig.json file before testing

### CODE REVIEW

File: localConfig_template.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "scheduleUpdate": {
+
+        "enabled": {
+            "@detail": {
+                "input_type": "option",
+                "options": {
+                    "true": true,
+                    "false": false
+                },
+                "skip_next_if": false
```
Comment (AdrienJarretier) [inline localConfig_template.json]: When configuring with scheduler disabled, if later this is changed to true, the time is undefined

I'm guessing this line is the issue.

**Review by AdrienJarretier — COMMENTED** [review]
Either your module could detect that the parameter changed and prompt for the rest of the configuration, I guessed that was the goal of https://github.com/AdrienJarretier/visualisation-cas-covid-19/blob/7bceb1c5e81e48c604ddb1b1eb71b57560a2b18b/localConfig_template.json#L20
either I was wrong or it doesn't work as intended ;

Or you could simple just ask for the schedule time anyway (before asking if we want it enabled).

**Comment by hugodecasta** [comment]
I understand, maybe later in a new version of the module, the system will re-read the config and ask for the right adjustements

File: localConfig_template.json (hunk 1 above)
Comment (hugodecasta) [reply localConfig_template.json]: Yes it is true ! It was made for a one time configuration, maybe I'll add a new feature to the module for reconfiguring.
I'll delete the line to force the scheduler configuration anyway

**Review by AdrienJarretier — APPROVED** [review]

**Pull request merged**